### PR TITLE
Update for 24-SHAPSHOT

### PR DESCRIPTION
**Pull request by davidpbauer**

### CODE REVIEW

**Comment by davidpbauer** [comment]
maven build is failing on Github with `Non-resolvable parent POM for org.sakaiproject.attendance:attendance:24-SNAPSHOT: Could not find artifact org.sakaiproject:master:pom:24-SNAPSHOT in sonatype-nexus-snapshots (https://oss.sonatype.org/content/repositories/snapshots)` is that expected or is there a way to resolve it?

File: impl/pom.xml [hunk 1]
```diff
@@ -18,10 +18,11 @@
...
 	<modelVersion>4.0.0</modelVersion>
 	<parent>
-    <artifactId>attendance</artifactId>
-    <groupId>org.sakaiproject.attendance</groupId>
-    <version>23-SNAPSHOT</version>
-  </parent>
+		<artifactId>attendance</artifactId>
+		<groupId>org.sakaiproject.attendance</groupId>
+		<version>24-SNAPSHOT</version>
+		<relativePath>../pom.xml</relativePath>
```
Comment (ottenhoff) [inline impl/pom.xml:24]: was this change purposeful?

Comment (davidpbauer) [reply impl/pom.xml:24]: It appeared to be common practice in Sakai to specify the relativePath to the parent module, so I added it in when updating the version to 24-SNAPSHOT. Then I figured it changed enough to warrant fixing the indentation for that section.

Comment (ottenhoff) [reply impl/pom.xml:24]: sorry, i was trying to suggest that maybe these changes have broken the build!

Comment (ottenhoff) [reply impl/pom.xml:24]: Is this build still failing?

**Pull request closed without merging**